dist/main/index.js

```diff
@@ -14536,8 +14537,15 @@ function ownerRepo(url) {
   };
 }
 
-async function genhtml(coverageFiles, tmpPath) {
-  const workingDirectory = core.getInput('working-directory').trim() || './';
+function ensureTrailingSlash(path) {
+  if (path.endsWith("/")) {
+    return path;
+  }
+
+  return path + "/";
+}
+
+async function genhtml(coverageFiles, tmpPath, workingDirectory) {
```

Comment:
minor: some linters would ding that this function was used before it was defined, ie, out of order